tools/ovc/openvino/tools/ovc/moc_frontend/type_utils.py

```diff
@@ -33,7 +33,22 @@ def to_ov_type(val):
     if 'tensorflow' in sys.modules:
         import tensorflow as tf # pylint: disable=import-error
         if isinstance(val, tf.dtypes.DType):
-            return Type(val.as_numpy_dtype())
+            tf_to_ov_type = {
+                tf.float32: ov.Type.f32,
+                tf.float16: ov.Type.f16,
+                tf.float64: ov.Type.f64,
+                tf.bfloat16: ov.Type.bf16,
+                tf.uint8: ov.Type.u8,
+                tf.int8: ov.Type.i8,
+                tf.int16: ov.Type.i16,
+                tf.int32: ov.Type.i32,
+                tf.int64: ov.Type.i64,
+                tf.bool: ov.Type.boolean,
+                tf.string: ov.Type.string
+            }
+            if val not in tf_to_ov_type:
+                raise Exception("The provided data time is not supported {}.".format(val))
+            return tf_to_ov_type[val]
```

Comment:
please do not introduce new dict of types. We should already created the same one in utils. Please check

Reply:
I am mistaked about existence. Let us create this dictionary in Python TF FE sources in utils.

Reply:
Done.